modules/tools/lab-partner/lab-partner-values.tpl

```diff
@@ -4,3 +4,9 @@ secrets:
   teamId: ${team_id}
 ingress:
   host: lab-partner.${namespace}.${cluster_domain}
+  annotations:
+    kubernetes.io/ingress.class: ${namespace}-nginx
```

Comment:
this ingress class should just be hardcoded to `toolchain-nginx`, or we should pass this in as a variable.  it isn't going to depend on the namespace that the lab partner is deployed to.